en/assembly/cable_wiring.md

```diff
@@ -0,0 +1,186 @@
+# Cable Wiring Basics
+
+Generally the cables are the main sources of [EMI](https://en.wikipedia.org/wiki/Electromagnetic_compatibility) troubles. Therefore many electrical hardware related failures could be avoid by proper cabling in the UAV. There are a few basics concepts related to RF induced failures to which are mandatory to mind during design of cabling in the drone.
+
+  * The High-Power and signaling cabling should be separated as much as is practical    
+  * The lengths of cabling should be reduced to amount necessary to easy handling of wired components. Tension of wires should be adequate to possible airframe deformations even in a crash landing. (The wires must not be the first to break)  
+  * Cable loops to reduce excess length needs to be avoided

```

Comment:
IMO it is worth identifying what problems might be caused. As far as I know the main one is EMI affecting the compasses resulting in poor directional control.

```suggestion
Cables are a common source of [electromagnetic interference (EMI)](https://en.wikipedia.org/wiki/Electromagnetic_interference), which can cause problems including flyways, "toilet bowling" and generally poor flight.
These problems can be avoided by using appropriate cabling
 in the UAV. 
 
The following basic concepts should be kept in mind when designing drone cabling:
* High-Power and signal cables should be separated as much as is practical    
* Cable lengths should be the minimum needed to enable easy handling of wired components.
  The wire tension should be adequate to survive possible airframe deformations even in a crash landing
  (wires must not be the first thing to break).
* Cable loops to reduce excess length should be avoided: use shorter lengths
```